src/base/system_parameter.c

```diff
@@ -728,6 +728,8 @@ static const char sysprm_ha_conf_file_name[] = "cubrid_ha.conf";
 
 #define PRM_NAME_STATDUMP_FORCE_ADD_INT_MAX "statdump_force_add_int_max"
 
+#define PRM_NAME_ORACLE_STYLE_DIVIDE "oracle_style_divide"
```

Comment:
Should we use the word "oracle"?
How about replacing it with words like "for math" or "for science",  "for engineering"?

Reply:
The purpose of this parameter is compatibility with oracle.